selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/ShareWorkspaceTest.java

```diff
@@ -0,0 +1,151 @@
+import static org.testng.Assert.assertTrue;
+
+import com.google.inject.Inject;
+import org.eclipse.che.commons.lang.NameGenerator;
+import org.eclipse.che.selenium.core.TestGroup;
+import org.eclipse.che.selenium.core.organization.InjectTestOrganization;
+import org.eclipse.che.selenium.core.organization.TestOrganization;
+import org.eclipse.che.selenium.core.user.AdminTestUser;
+import org.eclipse.che.selenium.core.user.TestUser;
+import org.eclipse.che.selenium.pageobject.dashboard.CheMultiuserAdminDashboard;
+import org.eclipse.che.selenium.pageobject.dashboard.NavigationBar;
+import org.eclipse.che.selenium.pageobject.dashboard.NewWorkspace;
+import org.eclipse.che.selenium.pageobject.dashboard.workspaces.WorkspaceDetails;
+import org.eclipse.che.selenium.pageobject.dashboard.workspaces.WorkspaceDetails.TabNames;
+import org.eclipse.che.selenium.pageobject.dashboard.workspaces.WorkspaceShare;
+import org.eclipse.che.selenium.pageobject.dashboard.workspaces.Workspaces;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+@Test(groups = {TestGroup.MULTIUSER})
+public class ShareWorkspaceTest {
+
+  private static final String WORKSPACE_NAME = generate("workspace", 4);
+  private static final String ADMIN_PERMISSIONS =
+      "read, use, run, configure, setPermissions, delete";
+  private static final String MEMBER_PERMISSIONS = "read, use, run, configure";
+
+  private String systemAdminName;
+  private String memberName;
+
+  @InjectTestOrganization private TestOrganization org;
+
+  @Inject private CheMultiuserAdminDashboard dashboard;
+  @Inject private WorkspaceDetails workspaceDetails;
+  @Inject private NavigationBar navigationBar;
+  @Inject private AdminTestUser adminTestUser;
+  @Inject private NewWorkspace newWorkspace;
+  @Inject private Workspaces workspaces;
+  @Inject private TestUser testUser;
+  @Inject private WorkspaceShare workspaceShare;
+
+  @BeforeClass
+  public void setUp() throws Exception {
+    org.addAdmin(adminTestUser.getId());
+    org.addMember(testUser.getId());
+    systemAdminName = adminTestUser.getEmail();
+    memberName = testUser.getEmail();
+
+    dashboard.open(adminTestUser.getName(), adminTestUser.getPassword());
+    createWorkspace(org.getName(), WORKSPACE_NAME);
+  }
+
+  @BeforeMethod
+  public void openShareWorkspaceTab() {
+    navigationBar.waitNavigationBar();
+
+    dashboard.selectWorkspacesItemOnDashboard();
+    workspaces.selectWorkspaceItemName(WORKSPACE_NAME);
+    workspaceDetails.waitToolbarTitleName(WORKSPACE_NAME);
+    workspaceDetails.selectTabInWorkspaceMenu(TabNames.SHARE);
+  }
+
+  @Test
+  public void checkShareWorkspaceTab() {
+    workspaceShare.waitMemberNameInShareList(systemAdminName);
+    assertEquals(workspaceShare.getMemberPermissions(systemAdminName), ADMIN_PERMISSIONS);
+
+    // check selecting member by checkbox
+    workspaceShare.clickOnMemberCheckbox(systemAdminName);
```

Comment:
Looks like a separate test, IMHO
I would prefer specialized tests to generalize ones.